client/src/main/java/com/alibaba/nacos/client/logging/AbstractNacosLogging.java

```diff
@@ -15,8 +15,9 @@
  */
 package com.alibaba.nacos.client.logging;
 
-import org.apache.commons.lang3.BooleanUtils;
-import org.apache.commons.lang3.StringUtils;
+
```

Comment:
remove extra blank line.

Reply:
In the lasted commit, the extra blank line has aleady been removed.